### ISSUE TITLE
If the profile exists for a credential's associated did, select the profile

### ISSUE THREAD

**Issue by kayaelle** [digitalcredentials/learner-credential-wallet#373]
From Pivotal list

Labels: 

status: rejected

created Aug 24, 2022

by Seth

Description: When adding a credential if the subject did already exists, only allow the user to add the credential to the profile with the corresponding subject did.

**Comment by dmitrizagidulin** [comment]
Right, so, I think what we should do here is _pre-select_ the existing profile (if it matches the incoming credential's subject.id). We shouldn't filter other profiles out, though, because a profile is more of a conceptual label -- you can store credentials in a profile that _aren't_ issued to that profile's DID

**Comment by kayaelle** [comment]
Is this issue about adding a credential manually?

@dmitrizagidulin - can you say more about "you can store credentials in a profile that aren't issued to that profile's DID"? Aren't profiles associated with DIDs? Do you mean that credentials without DIDs can be associated with any profile the user chooses?
